.github/pull_request_template.md

```diff
@@ -12,4 +12,7 @@ Describe how this change was tested.
 - [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
 - [ ] Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
```

Comment:
This is unrelated to the semver change. Just a thought before I forget, probably we can include 'Dependencies' too.
'Tag the PR with the type of update: **Bug**, **Enhancement**, **Dependencies** or **New-Feature**'

Reply:
Yep sure, I don't think we should be limited to these tags